src/styles/blocks/nav-link.scss

```diff
@@ -0,0 +1,18 @@
+.nav__link {
+  color: inherit;
+  text-decoration: none;
+  font-size: 22px;
+  font-weight: 700;
+  line-height: 22px;
+  letter-spacing: 2px;
+  text-align: left;
+}
```

Comment:
Should change on hover:
![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/c592b252-b1ae-453a-b0dc-66c7e182575a)

Also, remember to make this transition smooth!